src/tof/__init__.py

```diff
@@ -12,6 +12,7 @@
 del importlib
 
 from .chopper import AntiClockwise, Chopper, ChopperReading, Clockwise
+from .dashboard import Dashboard
```

Comment:
@MridulS would using `lazy_loader` fix the issue that this is effectively making `ipywidgets` a hard dependency of `tof`?
I would like to only have the `ImportError` if we try to use the dashboard...

Reply:
Yes, lazy_loader wouldn't eagerly load the dashboard so ImportError will only pop up once it is imported.